evap/student/templates/student_index_evaluation_period.html

```diff
@@ -0,0 +1,29 @@
+{% load evaluation_filters %}
+
+{% if evaluation.state == 'prepared' or evaluation.state == 'editor_approved' %}
+    <span class="small text-secondary" data-toggle="tooltip" data-placement="left"
+        title="{% trans 'The evaluation is currently prepared by the lecturers and the evaluation team.' %} {% trans 'You will receive an email when the evaluation period begins.' %}">
+        <i>{% trans 'In preparation' %}</i>
+    </span>
+{% elif evaluation.state == 'approved' %}
+    <span class="small{%if evaluation.hours_until_evaluation < 12 %} text-primary{% endif %}" data-toggle="tooltip" data-placement="left"
+        title="{% trans 'Evaluation period' %}:<br />{{ evaluation.vote_start_datetime }} &ndash; {{ evaluation.vote_end_date }}<br /><br />{% trans 'You will receive an email when the evaluation period begins.' %}">
+        <span class="far fa-calendar-alt"></span>
```

Comment:
that's definitely your screen :)
i don't like the plain calendar icon because i don't think it's clearly identifiable. with normal text size (which i also don't really like here) the squares look irregular on my screen, so i chose the clock icon. it fits to the icon now used for upcoming evaluations in the semester list.